connectivity/drivers/cellular/TELIT/ME310/TELIT_ME310_CellularStack.cpp

```diff
@@ -24,6 +24,12 @@
 static const int sslctxID = 1;
 
 using namespace mbed;
+using namespace std::chrono_literals;
+
+
+namespace mbed {
+constexpr auto ME310_SOCKET_TIMEOUT = 1s;
```

Comment:
```suggestion
constexpr auto me310_socket_timeout = 1s;
```
Because it is no longer a macro.